src/Fixer/StringNotation/SingleQuoteFixer.php

```diff
@@ -56,14 +73,27 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
 
             if (
                 '"' === $content[0] &&
-                false === strpos($content, "'") &&
+                (false === strpos($content, "'") || true === $this->configuration['strings_containing_single_quote_chars']) &&
```

Comment:
please swap the tests, the config check is cheaper than the `strpos`

Reply:
Done